src/dpdb/dpdb.js

```diff
@@ -19,7 +19,8 @@ var DPDB_CACHE = require('./dpdb-cache.js');
 var Util = require('../util.js');
 
 // Online DPDB URL.
-var ONLINE_DPDB_URL = 'https://storage.googleapis.com/cardboard-dpdb/dpdb.json';
+var ONLINE_DPDB_URL =
+  'https://thoragio.github.io/webvr-polyfill-dpdb/dpdb.json';
```

Comment:
if you want to transfer this to the [WebVRRocks](https://github.com/webvrrocks) org, I'd be happy to help you maintain this as well. if you want to keep it on your GH Pages, that's quite all right too 😄